File: week7/methodoverriding.cpp
#include<iostream>
using namespace std;

class Parent{
    public:
    void print(){
        cout << "This is a Parent method." << endl;
    }  
  
};

class Child : public Parent{
    public:
    void print(){
        
        cout << "This is a Child method." << endl;
    }
};

int main(){
    
    Child obj;
    obj.print();

}

File: week7/methodoverriding.java
class parent{
   public void display(){
       System.out.println("I'm in parent class.");
   }
}
class child extends parent{
   public void display(){
       super.display();
       System.out.println("I'm in child class.");
   }
}

public class methodoverriding {
   public static void main(String[] args) {
       child obj = new child();
       obj.display();

   }
}
